openapi2beans/go.sum

```diff
@@ -0,0 +1,70 @@
+github.com/cbroglie/mustache v1.4.0 h1:Azg0dVhxTml5me+7PsZ7WPrQq1Gkf3WApcHMjMprYoU=
```

Comment:
should this file even be checked-in ?

Reply:
im not sure. what does the cli project do...

Reply:
the cli was added for the factory originally I believe, will delete and regenerate file to see if it's still needed

Reply:
yes, it is used for the filesystem and console

Reply:
can copy and paste them into this project, but that's code duplication

Reply:
oops, completely misunderstood, the cli currently checks in the go sum